### PR TITLE
webble: add disconnect

### PR DESCRIPTION
**Pull request by jacobrosenthal**
Adds missing disconnect event to webble bindings.

### CODE REVIEW

File: lib/webbluetooth/bindings.js [hunk 1]
```diff
@@ -157,6 +157,13 @@ NobleBindings.prototype.connect = function(deviceUuid) {
   peripheral.device.gatt.connect()
     .then(function(gattServer){
       debug('peripheral connected', gattServer);
+
+      var onDisconnected = function(event){
+        debug('disconnected', event.target.id);
+        self.emit('disconnect', peripheral.uuid);
```
Comment (sandeepmistry) [inline lib/webbluetooth/bindings.js]: Should we remove the event listener here?

File: lib/webbluetooth/bindings.js [hunk 2]
```diff
@@ -157,6 +157,13 @@ NobleBindings.prototype.connect = function(deviceUuid) {
   peripheral.device.gatt.connect()
     .then(function(gattServer){
       debug('peripheral connected', gattServer);
+
+      var onDisconnected = function(event){
+        debug('disconnected', peripheral.uuid);
+        self.emit('disconnect', peripheral.uuid);
+      }
+      peripheral.device.addEventListener('gattserverdisconnected', onDisconnected, {once: true});
```
Comment (jacobrosenthal) [inline lib/webbluetooth/bindings.js]: @sandeepmistry to my knowledge thats handled with the {once: true} option

Comment (sandeepmistry) [inline lib/webbluetooth/bindings.js]: Ok, I missed that!

**Comment by sandeepmistry** [comment]
@monteslu if this looks good, please merge.

**Pull request merged**